packages/cardano-services/src/PgBoss/stakePoolMetricsHandler.ts

```diff
@@ -72,6 +72,14 @@ export const refreshPoolMetrics = async (options: RefreshPoolMetricsOptions) =>
   }
 };
 
+export const getPoolIdsToUpdate = async (dataSource: DataSource, outdatedSlot?: Cardano.Slot) =>
```

Comment:
Should this function provide TSDoc, as it is public (exported from the module)?

Do not know what are recommendations here or good practices.

Reply:
This function is only exposed for testing purposes. In our project, we are using index.ts files as an importing point. So no one should be directly 'importing' and using `getPoolIdsToUpdate`. Besides, I think the function name is explanatory enough for such a simple internal implementation. I could still and comments to explain it if you desire.

I noticed that we have these methods available in the index.ts. Removed them with f47a09fc4f